project-management/src/main/java/life/qbic/projectmanagement/application/ProjectOverview.java

```diff
@@ -58,6 +58,10 @@ public class ProjectOverview {
   @Column(name = "usernames")
   private List<String> collaboratorUserNames = new ArrayList<>();
 
+  @Convert(converter = CollaboratorUserInfosConverter.class)
+  @Column(name = "userInfos")
+  private List<UserInfo> collaboratorUserInfos = new ArrayList<>();
```

Comment:
This currently leads to an exception since the column does not exist in our database. 
Also the collaboratorUserNames field should be then removed here, from the specification and the sql view script to avoid redundant information imo.

Reply:
You have to use the data_management database 😉